build-support/setup_private_specs.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+# This script generates a private spec repo in your $HOME/.aws-amplify
+# directory, to enable `pod lib lint`. It also makes it easier to do
+# development against unreleased versions of Amplify, at least until we start
+# releasing nightly builds.
+
+set -e
+
+declare -r LOCAL_SPEC_GIT_ROOT="$HOME/.aws-amplify/aws-appsync-realtime-client-ios/aws-appsync-realtime-client-ios-podspecs.git"
+declare -r LOCAL_SPEC_REPO_NAME="aws-appsync-realtime-client-ios-local-specs"
+declare -r COCOAPODS_REPO_DIR="${HOME}/.cocoapods/repos/${LOCAL_SPEC_REPO_NAME}"
+
+function init_spec_git_root {
+  # If directory exists, assume it's correctly set up; don't attempt any
+  # repairs or validation
+  if [[ -d "${LOCAL_SPEC_GIT_ROOT}" ]] ; then
+    return 0
+  fi
+
+  mkdir -p "${LOCAL_SPEC_GIT_ROOT}"
+  old_pwd="${PWD}"
+  cd "${LOCAL_SPEC_GIT_ROOT}"
+  git init --bare
+  cd "${old_pwd}"
```

Comment:
I prefer:
```
mkdir -p "${LOCAL_SPEC_GIT_ROOT}"
pushd .
cd "${LOCAL_SPEC_GIT_ROOT}"
git init --bare
popd
```

But what you have also works.

Reply:
I'm not going to change this for now, since this is a nearly verbatim copy of the scripts from amplify & iOS SDK. Ideally, I'll move the script over to ci-support and parameterize it, so we can centralize these changes. But that's a job for future Tim. :)